tests/wpt-harness/viceroy.toml

```diff
@@ -0,0 +1,4 @@
+[local_server]
+  [local_server.backends]
+    [local_server.backends.wpt]
+      url = "http://web-platform.test:8000"
```

Comment:
Does this URL require any special changes to the /etc/hosts file? 😯

Reply:
it does, yes. Those changes happen [here](https://github.com/fastly/js-compute-runtime/pull/40/files#diff-7829468e86c1cc5d5133195b5cb48e1ff6c75e3e9203777f6b2e379d9e4882b3R150).